pandas/core/groupby/groupby.py

```diff
@@ -767,10 +767,23 @@ def __iter__(self) -> Iterator[tuple[Hashable, NDFrameT]]:
         """
         Groupby iterator.
 
+        This method provides an iterator over the groups created by the resampling
+        operation on the object. The method yields tuples where the first element
+        is the label (group key) corresponding to each resampled bin, and the second
+        element is the subset of the data that falls within that bin.
```

Comment:
This docstring is shared by groupby and resample, can you write this in a way that covers both of these cases?

Reply:
yes. on it.

Reply:
done